deps/fast_float/fast_float_strtod.cpp

```diff
@@ -0,0 +1,16 @@
+#include "fast_float.h"
+#include <iostream>
+#include <string>
+#include <system_error>
+
+/* Convert NPTR to a double using the fast_float library.
+ * If ENDPTR is not NULL, a pointer to the character after the last one used
+ * in the number is put in *ENDPTR.  */
+extern "C" double fast_float_strtod(const char *nptr, char **endptr) {
+  double result;
+  auto answer = fast_float::from_chars(nptr, nptr + strlen(nptr), result);
```

Comment:
If you want to catch potential errors, you ought to add..

```C++
 if(answer.ec != std::errc()) { /* do something */ }
```

(Note that the library follows the C++ standard so that's how we handle errors. We did not design this approach.)

Reply:
@lemire I've added a error reporting via errno same like strtod.